app/database_etl/postgres_tables_handler/table_formatter.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+import pandas as pd
+
+
+# Replace None utf-8 encoded characters with blank spaces
+def replace_null_string(x):
+    if type(x) != str:
+        return x
+    encoded_val = x.encode('utf-8')
+    if b'\x00' in encoded_val:
+        encoded_val = encoded_val.replace(b'\x00', b'\x20')
+        return encoded_val.decode('utf-8')
+    return x
+
+
+def add_mapped_variables(df):
```

Comment:
personally this function seems more related to the functions in `airtable_records_formatter` because it's applying transformations to the df in order to add/update columns

Reply:
ya, i see what you mean. i put it here because the order of which modules are used in sequential order is: airtable records to just get the records from airtable, then table handler to actually format those tables (which is where the mapped vars function comes in) and then postgres loader.